src/Esfa.Vacancy.Register.Api/App_Start/AutoMapperConfig.cs

```diff
@@ -2,40 +2,41 @@
 using AutoMapper;
 using Esfa.Vacancy.Register.Api.Mappings;
 using Esfa.Vacancy.Register.Application.Queries.SearchApprenticeshipVacancies;
+using StructureMap;
 using ApiTypes = Esfa.Vacancy.Api.Types;
 using DomainTypes = Esfa.Vacancy.Register.Domain.Entities;
 
 namespace Esfa.Vacancy.Register.Api
 {
     public static class AutoMapperConfig
     {
-        public static MapperConfiguration Configure()
+        public static MapperConfiguration Configure(IContext context)
         {
             return new MapperConfiguration(cfg =>
             {
+                cfg.ConstructServicesUsing(context.GetInstance);
```

Comment:
What does this do?

Reply:
This allows Automapper to resolve any dependencies from the container when creating new destination objects